src/grt/src/Grid.h

```diff
@@ -95,14 +89,14 @@ class Grid
 
   void clear();
 
-  long getLowerLeftX() const { return lower_left_x_; }
-  long getLowerLeftY() const { return lower_left_y_; }
+  long getLowerLeftX() const { return die_area_.xMin(); }
+  long getLowerLeftY() const { return die_area_.yMin(); }
 
-  void setLowerLeftX(long x) { lower_left_x_ = x; }
-  void setLowerLeftY(long y) { lower_left_y_ = y; }
+  void setLowerLeftX(long x) { die_area_.set_xlo(x); }
+  void setLowerLeftY(long y) { die_area_.set_ylo(y); }
 
-  long getUpperRightX() const { return upper_right_x_; }
-  long getUpperRightY() const { return upper_right_y_; }
+  long getUpperRightX() const { return die_area_.xMax(); }
+  long getUpperRightY() const { return die_area_.yMax(); }
```

Comment:
It would be nice if the access function name lined up with the odb function name. Not mandatory though.